docs/management/snapshot-restore/index.asciidoc

```diff
@@ -174,19 +199,106 @@ The repository currently doesn’t have any snapshots.
 Use the {ref}//modules-snapshots.html#snapshots-take-snapshot[snapshot API] to create a snapshot.
 
 . Go to *Dev Tools > Console*.
-. Create the snapshot. 
+. Create the snapshot by entering the following in the editor pane. 
++
+[source,js]
+PUT /_snapshot/my_backup/2019-04-25_snapshot
```

Comment:
I think this should be reformatted to easily copy/paste. It should be a single string: 

```PUT /_snapshot/my_backup/2019-04-25_snapshot?wait_for_completion=true```